pkg/controller/controller_utils.go

```diff
@@ -631,6 +672,86 @@ func maxContainerRestarts(pod *api.Pod) int {
 	return maxRestarts
 }
 
+// ControlActivePods first filters out inactive pods, then it tries to take
+// control of the remaining pods that matching the selector. It returns the pods
+// that are active and are controlled by the controller, and the pods that
+// should be abandoned by the controller.
+func ControlActivePods(podControl PodControlInterface,
```

Comment:
This function sends patch to take control of the pods, so I want to keep the "control" as a verb to emphasis this fact. Let's continue the discussion in #27600.